packages/react-components/src/components/table-output-component.tsx

```diff
@@ -74,9 +81,9 @@ export class TableOutputComponent extends AbstractOutputComponent<TableOutputPro
                 onGridReady={this.onGridReady}
                 components={this.components}
                 onCellClicked={this.onEventClick}
-                rowSelection='single'
+                rowSelection='multiple'
```

Comment:
Two issues with table selection:

- When pressing space bar it adds the row to current selection, same as Ctrl+click. This might be the default 'multiple' behaviour. We might want to only allow contiguous selection. In Eclipse TC, Ctrl+click replaces the selection, maybe should be the same, also for space bar.
- When using the scroll bar to go far enough that model is updated, and the current selection index is out of range of the current virtual window, then it is not possible to extend selection with Shift+click. You need to start a new selection with normal click first before you can extend it again.

Reply:
I have made changes for the first scenario. Both space and ctrl+click now has the right behavior. They only select a single node and deselects all the previously selected nodes. 

Should we handle the second scenario explicitly? The grid is supposed to handle it by default.

Reply:
I fixed both the issues now. could you please review the latest. Thanks.